language-plutus-core/test/Evaluation/ApplyBuiltinName.hs

```diff
@@ -1,27 +1,42 @@
 -- | Constant application tests.
 
-{-# LANGUAGE GADTs      #-}
-{-# LANGUAGE RankNTypes #-}
+{-# OPTIONS_GHC -fno-warn-orphans #-}
+
+{-# LANGUAGE FlexibleInstances     #-}
+{-# LANGUAGE GADTs                 #-}
+{-# LANGUAGE MultiParamTypeClasses #-}
+{-# LANGUAGE RankNTypes            #-}
+{-# LANGUAGE TypeApplications      #-}
 
 module Evaluation.ApplyBuiltinName
     ( test_applyBuiltinName
     ) where
 
 import           Language.PlutusCore
 import           Language.PlutusCore.Constant
-import           Language.PlutusCore.Evaluation.Machine.Ck
+import           Language.PlutusCore.Evaluation.Machine.ExBudgeting
+import           Language.PlutusCore.Evaluation.Machine.ExBudgetingDefaults
+import           Language.PlutusCore.Evaluation.Machine.Exception
+import           Language.PlutusCore.Evaluation.Machine.ExMemory
 import           Language.PlutusCore.Generators
 import           Language.PlutusCore.Pretty
 
-import qualified Data.ByteString.Lazy                      as BSL
-import qualified Data.ByteString.Lazy.Hash                 as Hash
+import qualified Data.ByteString.Lazy                                       as BSL
+import qualified Data.ByteString.Lazy.Hash                                  as Hash
 import           Data.Coerce
 import           Data.Foldable
 import           Data.List
-import           Hedgehog                                  hiding (Var)
+import           Hedgehog                                                   hiding (Var)
 import           Test.Tasty
 import           Test.Tasty.Hedgehog
 
+type NoBudgetingM = Either (EvaluationException (Plain Term DefaultUni) () ())
```

Comment:
Note to self: if removing `ExMemory` from the CK machine works out, the CK should be used in this module as before.